lib/bumper.js

```diff
@@ -212,17 +218,28 @@ class Bumper {
       // the --oneline format for `git log` puts each commit on a single line, with the hash and then
       // the commit message, so we first split on \n to get an array of commits
       const commits = stdout.split('\n')
+      let prNumber
 
       // The commit that represents the merging of the PR will include the text 'Merge pull request' so
       // we find that one
-      const mergeCommit = __.find(commits, (commit) => {
+      let mergeCommit = __.find(commits, (commit) => {
         return commit.indexOf('Merge pull request') !== -1
       })
 
-      // The format of the auto-generated commit line will look something like:
-      // 'edf85e0 Merge pull request #30 from job13er/remove-newline'
-      // so if we split on the space, and grab the 5th item, it's '#30' then strip the '#' to get '30'
-      const prNumber = mergeCommit.split(' ')[4].replace('#', '')
+      if (mergeCommit) {
+        // The format of the auto-generated commit line will look something like:
+        // 'edf85e0 Merge pull request #30 from job13er/remove-newline'
+        // so if we split on the space, and grab the 5th item, it's '#30' then strip the '#' to get '30'
+        prNumber = mergeCommit.split(' ')[4].replace('#', '')
+      } else {
+        // Bitbucket cloud has it's own format for PR merge messages.
+        // The commit that represents the merging of the PR will include the text 'pull request #' so
+        // we find that one
+        mergeCommit = __.find(commits, (commit) => {
+          return commit.match(/pull request #/) !== null
```

Comment:
I'm a little confused about how this is different, the example for non bitbucket cloud also includes the text `pull request #`, couldn't we use the same code for both cases?

Reply:
Good point, I think we can make a clever regex that handles both - I'll give it a look.

Reply:
I think your existing regex may even work as-is :)

Reply:
yep, reduced and removed specific vcs callouts.